### PR TITLE
[DOCU-2783] Plugin ordering in Kong Manager

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

Added instructions for how to configure plugin ordering with Kong Manager. This is for Gateway 3.1. 
### Reason

DOCU-2783
### Testing

Go to: KM tabs in `/gateway/latest/kong-enterprise/plugin-ordering/get-started/`

Questions for writer reviewers:

- I originally created a "Plugin Ordering" subsection in the Kong Manager section of the docs for this info, but I was mostly copying what was in the Kong Enterprise "Plugin Ordering" subsection, so I figured I'd just add it there. Does this work, or should this live in Kong Manager? I thought maybe at one point we ripped out KM instructions during 3.0 and moved them to the KM section.
- I know this is documenting the UI, but I think it's okay here because it's focused on teaching the user how plugin ordering works with an example vs. generic plugin ordering instructions. Thoughts?

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@Leopoldthecoder Hi! Could you review the changes I made in this PR for technical accuracy? Here's the link to the doc preview: https://deploy-preview-4790--kongdocs.netlify.app/gateway/latest/kong-enterprise/plugin-ordering/get-started/

I also had a few questions:

- Is this for enterprise only, or is this feature free? 
- Are all the steps correct for Kong Manager? I looked at the configurations for the API and yaml to figure out what fields in Kong Manager would need to be configured, but I just wanted to double check.

**Comment by Leopoldthecoder** [comment]
@cloudjumpercat Hi!
> Is this for enterprise only, or is this feature free?

This is for enterprise only.

##

> Are all the steps correct for Kong Manager?

Yes, you got them right.🚀

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

File: src/gateway/kong-enterprise/plugin-ordering/get-started.md [hunk 1]
```diff
@@ -130,6 +130,28 @@ ordering:
...
+6. Click **Install**.
+7. From the **Rate Limiting** plugin page, click the **Ordering** tab.
+8. Click **Add ordering**.
+9. For **Before access**, click **Add plugin**.
+10. Choose **Key Auth** from the **Plugin 1** dropdown menu.
+11. Click **Update**.
+
+The rate limiting plugin now limits the amount of requests against all services and routes in the default workspace *before* {{site.base_gateway}} requests authentication.
+
 {% endnavtab %}
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/plugin-ordering/get-started.md:155]: Have you tried surrounding the navtabs with an `{% if_version %}` tag? In this case, that would be surrounding lines 134-156. Not sure if it works, worth a try.

**Review by lena-larionova — CHANGES_REQUESTED** [review]
> I know this is documenting the UI, but I think it's okay here because it's focused on teaching the user how plugin ordering works with an example vs. generic plugin ordering instructions. Thoughts?

I agree that for examples or tutorials, UI instructions make sense to include. 

The new tabs need to be conditionalized, otherwise this looks good.

File: src/gateway/kong-enterprise/plugin-ordering/get-started.md (hunk 1 above)
Comment (cloudjumpercat) [reply src/gateway/kong-enterprise/plugin-ordering/get-started.md]: @lena-larionova I tried adding the conditional rendering, but it wasn't working locally. Not sure if I did something wrong or if it's just not possible to conditionalize a tab.

Comment (lena-larionova) [reply src/gateway/kong-enterprise/plugin-ordering/get-started.md]: Looks like it's not possible :( we'll have to ask Michael or Fabian for support on that.

For now, you could just add a note in the KM tabs that says something like "Kong Manager support for dynamic plugin ordering is available starting in Kong Gateway 3.1"

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**